cmd/server/main.go

```diff
@@ -26,6 +26,11 @@ func main() {
 			Value: "/var/lib/ocid/sandbox",
 			Usage: "ocid pod sandbox dir",
 		},
+		cli.StringFlag{
+			Name:  "runtime",
+			Value: "/usr/bin/runc",
```

Comment:
@runcom I think giving a path is more deterministic so we don't have to worry about the PATH and what binary may be picked up by the daemon.